public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -1256,4 +1266,15 @@ else if(Integer.parseInt(ard.getStratum3()) == '8') {
             aa.addResult(achillesResult);
         }
     }
+
+    @Async("asyncExecutor")
+    public Future<String> searchTrackEvent(String category, String event, String label) {
+        int response = -1;
+        try {
+            response = googleAnalyticsService.trackEventToGoogleAnalytics(null, category, event, label);
+        } catch (IOException ioException) {
+            response = -1;
```

Comment:
This should be redundant with line 1272. `trackEventToGoogleAnalytics` cannot both return a value and throw an exception

I would also suggest logging a warning here, or this information will be lost